### PR TITLE
add JSON data into email

### PR DESCRIPTION
**Pull request by KochTobi**
attach data into the email below a defined attachment header.

### CODE REVIEW

File: _includes/data/access.html [hunk 1]
```diff
@@ -29,10 +29,12 @@ <h2>Request Access</h2>
 					<label for="institution">Institution:</label>
 					<input type="textfield" name="institution" class="form-control" id="institution" placeholder="University of New Orleans" required=true>
 				</div>
-				<div class="form-group">
+				<!--
```
Comment (vinjana) [inline _includes/data/access.html]: Can probably be deleted.

File: _includes/data/access.html [hunk 2]
```diff
@@ -114,8 +126,9 @@ <h2>Request Access</h2>
 
 ${attachmentHeader}
 ${attachment}`
-		subject = `Access Request on ${datasets}`;
+		subject = `Access Request on ${datasetIds}`;
 		try {
+			document.getElementById('form-output').innerHTML = `<textarea class="form-control" rows="15" disabled>${body}</textarea>`;
```
Comment (vinjana) [inline _includes/data/access.html]: The body does not contain the target email address. So the user may not know where to send the text to.

As the sending feature works, I think this is not so important now.

**Comment by vinjana** [comment]
I noticed that it may be good to write something like

The following email was sent to ghga... You will receive a confirmation email with a ticket number in the next minutes.

This would make clear, that it is not expected that the email will be copied with a stupid browser configuration like mine (just send the mail, without asking back). 

The mail will probably be send to our ticket system, which automatically creates a new ticket and informs the requester.

File: _includes/data/access.html (hunk 1 above)
Comment (KochTobi) [reply _includes/data/access.html]: Yes

**Pull request merged**